lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -65,15 +66,15 @@ sub _get_database {
     my ($request) = @_;
     my $creds = $request->{_auth}->get_credentials('setup');
 
-    return [ 401,
+    return [ HTTP_UNAUTHORIZED,
              [ 'WWW-Authenticate' => 'Basic realm="LedgerSMB"',
                'Content-Type' => 'text/text; charset=UTF-8' ],
              [ 'Please enter your credentials' ] ]
         if ! defined $creds->{password};
 
     # Ideally this regex should be configurable per instance, and possibly per admin user
     # for now we simply use a fixed regex. It will cover many if not most use cases.
-    return [ 454,
+    return [ HTTP_454,
```

Comment:
in light of other uses of 454 I think this should change to be....
use constant { HTTP_454_PROHIBITED_USERNAME => '454 Prohibited Username', };

As we use 454 to present error's directly to our client UI for display